docs/development.md

```diff
@@ -59,22 +59,30 @@ bundle exec foreman start
 
 ### Stable Katello Development Box
 
-A stable box image of a fully installed Katello development environment is published to Vagrant cloud nightly (on successful installs). In the `vagrant/boxes.d/99-local.example` file, a box using this image is available under the name `centos7-katello-devel-stable`. If it's the first time spinning this box up, you can do:
+#### Using the stable Katello development box
 
-```
-vagrant up centos7-katello-devel-stable
-```
+When spinning up a Katello development environment locally, it can take a while to install and isn't always guaranteed finish successfully. A stable box image of a fully installed Katello development environment is created nightly and [is published to Vagrant cloud](https://app.vagrantup.com/katello/boxes/katello-devel) when the install succeeds. This image will have a Katello development environment successfully installed, ready to use.
```

Comment:
guaranteed to finish successfully.

Reply:
I would note somewhere if we are not already that the box will need to be setup and configured for the users environment. Since the normal box when spun up sets up user remotes for example and this one does not.

Reply:
Well, I see that line further below. But I would move that comment up to this section.